src/OpenColorIO/transforms/builtins/ACES.cpp

```diff
@@ -598,26 +843,190 @@ void RegisterAll(BuiltinTransformRegistryImpl & registry) noexcept
+
+            CreateRangeOp(ops, 
+                          RangeOpData::EmptyValue(), 0.918,  // don't clamp low end
+                          RangeOpData::EmptyValue(), 0.918,  // don't clamp low end
+                          TRANSFORM_DIR_FORWARD);
+
+            static constexpr double scale     = 0.96;
+            static constexpr double scale4[4] = { scale, scale, scale, 1. };
+            CreateScaleOp(ops, scale4, TRANSFORM_DIR_FORWARD);
+
+            MatrixOpData::MatrixArrayPtr matrix
+                = rgb2xyz_from_xy(ACES_AP1::primaries);
+            CreateMatrixOp(ops, matrix, TRANSFORM_DIR_FORWARD);
+
+            MatrixOpData::MatrixArrayPtr matrix2
+                = build_vonkries_adapt(WHITEPOINT::DCI_XYZ, WHITEPOINT::D65_XYZ, ADAPTATION_BRADFORD);
+            CreateMatrixOp(ops, matrix2, TRANSFORM_DIR_FORWARD);
+        };
+
+        registry.addBuiltin("ACES-OUTPUT--ACES_to_CIE-XYZ_SDR-CINEMA-D60sim-DCI_1.0", 
+                            "Component of ACES Output Transforms for SDR DCI cinema simulating D60 white",
+                            ACES2065_1_to_CIE_XYZ_cinema_d60sim_dci_1_0_Functor);
+    }
+
+    {
+        auto ACES2065_1_to_CIE_XYZ_cinema_d65sim_dci_1_1_Functor = [](OpRcPtrVec & ops)
+        {
+            ACES_OUTPUT::Generate_RRT_preamble_ops(ops);
+
+            ACES_OUTPUT::Generate_tonecurve_ops(ops);
+
+            ACES_OUTPUT::Generate_roll_white_d65_ops(ops);
+
+            CreateRangeOp(ops, 
+                          RangeOpData::EmptyValue(), 0.908,  // don't clamp low end
+                          RangeOpData::EmptyValue(), 0.908,  // don't clamp low end
+                          TRANSFORM_DIR_FORWARD);
+
+            static constexpr double scale     = 0.9575;
+            static constexpr double scale4[4] = { scale, scale, scale, 1. };
+            CreateScaleOp(ops, scale4, TRANSFORM_DIR_FORWARD);
+
+            AP1_to_CIE_XYZ_D65::GenerateOps(ops);
+
+            MatrixOpData::MatrixArrayPtr matrix2
+                = build_vonkries_adapt(WHITEPOINT::DCI_XYZ, WHITEPOINT::D65_XYZ, ADAPTATION_BRADFORD);
+            CreateMatrixOp(ops, matrix2, TRANSFORM_DIR_FORWARD);
+        };
+
+        registry.addBuiltin("ACES-OUTPUT--ACES_to_CIE-XYZ_SDR-CINEMA-D65sim-DCI_1.1", 
+                            "Component of ACES Output Transforms for SDR DCI cinema simulating D65 white",
+                            ACES2065_1_to_CIE_XYZ_cinema_d65sim_dci_1_1_Functor);
+    }
+
+    {
+        auto ACES2065_1_to_CIE_XYZ_hdr_video_1000nits_rec2020lim_1_1_Functor = [](OpRcPtrVec & ops)
+        {
+            ACES_OUTPUT::Generate_RRT_preamble_ops(ops);
+
+            ACES_OUTPUT::Generate_hdr_tonecurve_ops(ops, 1000.);
+
+            ACES_OUTPUT::Generate_hdr_primary_clamp_ops(ops, REC2020::primaries);
+
+            ACES_OUTPUT::Generate_nit_normalization_ops(ops, 1000.);
+        };
+
+        registry.addBuiltin("ACES-OUTPUT--ACES_to_CIE-XYZ_HDR-VIDEO-1000nit-15nit-REC2020lim_1.1", 
```

Comment:
Personally, I find that some new builtin names are less and less readable. A suggestion:
`ACES-OUTPUT / ACES2065_1__to__CIE-XYZ_HDR-VIDEO-1000nit-15nit-REC2020lim_1.1`. But the last part is still 'dense'!

And the explanation does not seem to contain all the information (as in the builtin name)